src/Sulu/Bundle/PageBundle/DependencyInjection/SuluPageExtension.php

```diff
@@ -102,6 +102,14 @@ public function prepend(ContainerBuilder $container)
                 [
                     'content' => [
                         'structure' => [
+                            'required_properties' => [
+                                'home' => ['title'],
+                                'page' => ['title'],
+                            ],
+                            'required_tags' => [
+                                'home' => ['sulu.rlp'],
+                                'page' => ['sulu.rlp'],
```

Comment:
is not `sulu.rlp.part` required and `sulu.rlp` should not longer be needed: https://github.com/sulu/sulu/blob/develop/UPGRADE.md#sulurlp-tag-deprecated

Reply:
fixed

Reply:
@wachterjohannes 😕